ui/src/features/common/DetailsBox.tsx

```
@@ -86,38 +85,28 @@
+            <div className="flex flex-col gap-2">
+              {filteredContent.map(({ value, type }) => (
                 <p
-                  className={`${detail.type === "link" ? "cursor-pointer font-light text-blue-400 hover:text-blue-300" : "font-normal"}`}
+                  className={`${type === "link" ? "cursor-pointer font-light text-blue-400 hover:text-blue-300" : "font-light"}`}
                   onClick={
-                    detail.type === "link"
-                      ? openLinkInNewTab(detail.value)
+                    type === "link" && value
+                      ? openLinkInNewTab(value)
                       : undefined
                   }>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)